From 56a75be530e26bf19f730957d10e46b6c6ccf609 Mon Sep 17 00:00:00 2001
From: Kathy Brade <brade@pearlcrescent.com>
Date: Tue, 25 Oct 2016 23:01:20 -0400
Subject: [PATCH 7/7] fixup! Bug 1070710 - Add mozilla::ViewRegion which
 assembles a LayoutDeviceIntRegion as NSViews. r=spohl

OSX: Fix a problem where clicking and dragging in the content area of a
window would sometimes cause the window to move. This is a fixup for an
error introduced while backporting a Mozilla patch for Tor bug 20204.
---
 widget/cocoa/ViewRegion.mm | 1 -
 1 file changed, 1 deletion(-)

diff --git a/widget/cocoa/ViewRegion.mm b/widget/cocoa/ViewRegion.mm
index 3459849..ee31889 100644
--- a/widget/cocoa/ViewRegion.mm
+++ b/widget/cocoa/ViewRegion.mm
@@ -56,7 +56,6 @@ ViewRegion::UpdateRegion(const LayoutDeviceIntRegion& aRegion,
       }
       [view setNeedsDisplay:YES];
       mViews.AppendElement(view);
-      iter.Next();
     } else {
       // Our new region is made of fewer rects than the old region, so we can
       // remove this view. We only have a weak reference to it, so removing it
-- 
2.10.1

